test/test_ecs_utils.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from unittest import mock
+from dcicutils.ecs_utils import ECSUtils
+
+
+@pytest.fixture(scope='module')
+def ecs_utils():
+    return ECSUtils(cluster_name='dummy-cluster')
+
+
+def mock_update_service(*, cluster, service, forceNewDeployment):
+    """ Mock matching the relevant API signature for below (we don't actually want
```

Comment:
Sometime we should discuss this doc string indentation style and why it bugs me. But it's OK for now.